app/assets/javascripts/blacklight/facet_load.js

```diff
@@ -0,0 +1,15 @@
+(function($) {
+  Blacklight.onLoad(function() {
```

Comment:
Ok so figured out the answer for this.. @jkeck @cbeer 

You can do this at the top of the file:

```javascript
/*global Blacklight */
```